fr/translation.json

```diff
@@ -572,5 +572,6 @@
   "high_output": "High Output",
   "felt_to_uint384": "Felt to uint384",
   "mid_output": "Mid Output",
-  "this_contract_not_balances": "This contract does not have token balances"
+  "this_contract_not_balances": "This contract does not have token balances",
+  "gas_price": ""
```

Comment:
in case of no transalation, leave it as english so other people can do it later.

Reply:
oh i think i made the translation but it did not appear on git (maybe some character set issues?) nvm I'll leave them as English version